src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  display: flex;
+  margin: 10px;
```

Comment:
It's not a good practice to use margins on BEM blocks. Consider moving it to the element or modifier that needs it.